nCompiler/R/developerTools.R

```diff
@@ -1,5 +1,50 @@
 nErrorEnv <- new.env()
+nDebugEnv <- new.env()
 
 ## Set nErrorEnv$stateInfo to cache state information (character)
 ## for display in the event of an error.
 
+getCompilerLog <- function(echo = FALSE) {
```

Comment:
I like the style of writing these generic functions.  This will allow us to modify the style of the logging done by each of these relatively easily.